pages/public_cloud/containers_orchestration/managed_kubernetes/forwarding-audit-logs-to-logs-data-platform/guide.en-gb.md

```diff
@@ -0,0 +1,197 @@
+   }
+}
+```
+
+## How to enable Audit Log Forwarding
+### Using the Manager
+
+#### Create and see existing streams
+
+ You can find your streamId, in Logs Data Platform control panel of your account in Data stream” pan when editing a stream:
+
+ -	Got to the “Data Stream” page of your Logs Data Platform account and “Edit” the target Data Stream
+
+ https://www.ovh.com/manager/#/dedicated/dbaas/logs/{ldp-account-id}/streams/home
+
+ -	Copy the streamId page of your Logs Data Platform account.
+
+ https://www.ovh.com/manager/#/dedicated/dbaas/logs/{ldp-account-id}/streams/{streamid}/edit
+
+### Using APIs
+
+You will have to define the targeted *Stream* of one of your LDP account on which you want your logs to be forwarded to. The enablement of the forwarding will create a subscription for this stream id.
+Note that the forwarding activation is free of charge, but you will be charged for the usage of Logs Data Platform service as per standard price plan. For LDP pricing refer to [Logs Data Platform product page](https://www.ovhcloud.com/en-ie/logs-data-platform/).
+
+You can retrieve the API specification in [OVH API Portal](https://eu.api.ovh.com/console-preview/?section=%2Fdbaas%2Flogs&branch=v1#post-/dbaas/logs/-serviceName-/output/graylog/stream)
+
+###### 1/ Retrieve your target Stream (and ID):
+
+List data stream of you Logs Data Platform account ([API reference](https://eu.api.ovh.com/console-preview/?section=%2Fdbaas%2Flogs&branch=v1#get-/dbaas/logs/-serviceName-/output/graylog/stream)):
+
+```shell
+GET /dbaas/logs/{serviceName}/output/graylog/stream
+```
+Get the details of a data stream ([API reference](https://eu.api.ovh.com/console-preview/?section=%2Fdbaas%2Flogs&branch=v1#get-/dbaas/logs/-serviceName-/output/graylog/stream/-streamId-)):
+```shell
+GET   /dbaas/logs/{serviceName}/output/graylog/stream/{streamId}
+```   
+###### 2/ Create your subscription
+
+As per below example, the POST request has a payload containing a streamId, which is the target data stream of your LDP account where you aim your Kubernetes cluster Audit logs will be forwarded. You also need to specifiy the 'kind' of log you want to forward, note that currenlty the only supported value for Managed Kubernetes Service is 'audit' (you can find available kinds using the [dedicated API](https://eu.api.ovh.com/console-preview/?section=%2Fcloud&branch=v1#get-/cloud/project/-serviceName-/capabilities/kube/log/kind) call) :
+```shell
+POST /cloud/project/{serviceName}/kube/{kubeId}/log/subscription
+{
+  "kind": "string",   //Currently the only supported value is 'audit'
+  "streamId": "18d602ec-af40-4000-8e59-41ecc8c23f80" //the streamID of the targeted Stream
+}
+```
+
+You will get in response an operationId
+```shell
+{
+"operationId": "18d3aecd-bae0-4000-8558-4b60e496bb80",
+"serviceName": "string"
+}
+```
+
+So you can use it to retrieve the subscriptionid further management purposes using Logs Data Platform read operation endpoint:
```

Comment:
subscriptionid → `subscriptionId`

Here you describe how to use the `operationId` to get you `subscriptionId`, but you directly give the MKS endpoint. That's confusing

Instead you can describe how to follow this operation through [LDP operation route](https://eu.api.ovh.com/console-preview/?section=%2Fdbaas%2Flogs&branch=v1#get-/dbaas/logs/-serviceName-/operation/-operationId-), which will give you the `subscriptionId` once succeed

And then continue wit this `GET /cloud/project/{serviceName}/kube/{kubeId}/log/subscription/{subscriptionId}` description